editor/src/clj/editor/bundle_dialog.clj

```diff
@@ -250,41 +250,46 @@
      (ui/add-style! "toggles")
      (ui/children! [(doto (CheckBox. "Generate debug symbols") (.setId "generate-debug-symbols-check-box") (.setFocusTraversable false) (ui/on-action! refresh!))
                     (doto (CheckBox. "Generate build report") (.setId "generate-build-report-check-box") (.setFocusTraversable false) (ui/on-action! refresh!))
-                    (doto (CheckBox. "Publish Live Update content") (.setId "publish-live-update-content-check-box") (.setFocusTraversable false) (ui/on-action! refresh!))]))])
+                    (doto (CheckBox. "Publish Live Update content") (.setId "publish-live-update-content-check-box") (.setFocusTraversable false) (ui/on-action! refresh!))
+                    (doto (CheckBox. "Bundle as build target") (.setId "bundle-as-build-target-check-box") (.setFocusTraversable false) (ui/on-action! refresh!))]))])
```

Comment:
@britzl also I'm not sure about this text for the end user in the editor 
`"Bundle as build target"`

Reply:
"Bundle application without resources" ?

Reply:
It's tricky! Is "contentless" a good term, perhaps?